src/core/src/op/paged_attention.cpp

```diff
@@ -8,6 +8,88 @@
 #include "itt.hpp"
 #include "openvino/op/op.hpp"
 
+namespace {
+
+using namespace ov;
```

Comment:
Consider move it inside `ov::op` namespace using should not be required.

Reply:
move using-s inside the implemented helper